src/rotate.c

```diff
@@ -0,0 +1,26 @@
+#include "rotate.h"
+
+void rotate_block(Block* b, const int rotation_matrix[2][2]){
+    int rotated_cells[4][2];
+
+    for (int i=0;i<4;i++) {     //row of first matrix
```

Comment:
Can you add spaces after semicolons. for consistency. here and the two for loops below as well